### PR TITLE
Feat/add cmd pull

### PR DESCRIPTION
**Pull request by jycouet**

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -115,47 +124,51 @@ async function main() {
...
-	}
 
-	p.outro(`🎉 Everything is ready!`);
+		p.outro(`🎉 Everything is ready!`);
+	}
 
 	process.exit(0);
 }
 
+function pull() {}
```
Comment (Yedidyar) [inline src/cli.ts:170]: Do forgot it?

**Review by Yedidyar — CHANGES_REQUESTED** [review]

File: src/cli.ts (hunk 1 above)
Comment (jycouet) [reply src/cli.ts:170]: yes ^^

**Review by Yedidyar — APPROVED** [review]

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# DATABASE_URL = "postgres://postgres:example@127.0.0.1:5432/my-db"
```
Comment (Yedidyar) [inline .env.example:1]: do you want to uncomment it

Comment (jycouet) [reply .env.example:1]: No no, I just want to show it here

**Pull request merged**